packages/frontend/src/features/sqlRunner/components/visualizations/BarChart.tsx

```diff
@@ -0,0 +1,34 @@
+import EChartsReact from 'echarts-for-react';
+import { type EChartsReactProps } from 'echarts-for-react/lib/types';
+import { memo, type FC } from 'react';
+import { type useSqlQueryRun } from '../../hooks/useSqlQueryRun';
+import { useAppSelector } from '../../store/hooks';
+import { useBarChartDataTransformer } from '../../transformers/useBarChartDataTransformer';
+
+type BarChartProps = Omit<EChartsReactProps, 'option'> & {
+    data: NonNullable<ReturnType<typeof useSqlQueryRun>['data']>;
+};
+
+const BarChart: FC<BarChartProps> = memo(({ data, className, ...rest }) => {
+    // TODO: fix store type
+    const barChartConfig = useAppSelector(
+        (state: any) => state.sqlRunner.chartConfig,
```

Comment:
```suggestion
        (state) => state.sqlRunner.chartConfig,
```

Reply:
And actually we can move back the type of 
```
chartConfig: BarChartConfig | undefined;
```
to just be

```
chartConfig: BarChartConfig;
```

makes sense what you had before, sorry